app/_src/using-mesh/managing-ingress-traffic/gateway-api.md

```diff
@@ -293,28 +293,123 @@ Gateway API isn't supported with multi-zone deployments, use {{site.mesh_product
+            end
+        end
+    end
+    subgraph c1["k8s-cluster-1"]
+        subgraph z1["zone-1"]
+            subgraph Gateway
+                listener(:8080)
+            end
+            subgraph Service
+                b1(backend)
+            end
+        end
+    end
+{% endmermaid %}
 
 If you deploy multi-zone with Gateway API, the following will occur:
-   
-   - If you create an `HTTPRoute` with a `backendRef` targeting the backend service in `zone-1`, it will only route traffic to the `backend` service in `zone-1`.
-   - Similarly, if you create an `HTTPRoute` in `zone-2` with a `backendRef` pointing to the `db` service, it will result in a `HTTPRoute` with a `ResolvedRefs` status condition of `BackendNotFound` because service `db` is not present in `zone-1`.
+
+{% capture gapi_multizone_limitation_1 %}
+{% mermaid %}
+flowchart TD
+    subgraph c2["k8s-cluster-2"]
+        subgraph z2["zone-2"]
+            subgraph c1z2s1["Service"]
+                backend2(backend)
+            end
+            subgraph c1z2s2["Service"]
+                db(db)
+            end
+        end
+    end
+    subgraph c1["k8s-cluster-1"]
+        subgraph z1["zone-1"]
+            subgraph Service
+                backend1(backend)
+            end
+            subgraph Gateway
+                listener(:8080)
+            end
+            subgraph HTTPRoute
+                route1(/)
+            end
+            route1--"❌"-->backend2
+            linkStyle 0 stroke:red,color:red,stroke-dasharray: 5 5;
+            route1-->backend1
+            listener-->route1
+        end
+    end
+{% endmermaid %}
+{% endcapture %}
+
+{% capture gapi_multizone_limitation_2 %}
+{% mermaid %}
+flowchart TD
+    subgraph c2["k8s-cluster-2"]
+        subgraph z2["zone-2"]
+            subgraph c1z2s1["Service"]
+                backend2(backend)
+            end
+            subgraph c1z2s2["Service"]
+                db(db)
+            end
+        end
+    end
+    subgraph c1["k8s-cluster-1"]
+        subgraph z1["zone-1"]
+            subgraph Gateway
+                listener(:8080)
+            end
+            subgraph HTTPRoute
+                route1(/)
+            end
+            subgraph Service
+                backend1(backend)
+            end
+            route1--"❌"-->db
+            linkStyle 0 stroke:red,color:red,stroke-dasharray: 5 5;
+            listener-->route1
+        end
+    end
+{% endmermaid %}
+{% endcapture %}
+
+   - If in `k8s-cluster-1` you create an `HTTPRoute` with a `backendRef` targeting the `backend` service, it will only route traffic to the `backend` service in `k8s-cluster-1`.
+     
+     {{ gapi_multizone_limitation_1 | indent }}
+
+   - Similarly, if in `k8s-cluster-1` you create an `HTTPRoute` with a `backendRef` pointing to the `db` service, it will result in a `HTTPRoute` with a `ResolvedRefs` status condition of `BackendNotFound` because service `db` is not present in `k8s-cluster-1`.
```

Comment:
```suggestion
   - Similarly, if you create an `HTTPRoute` with a `backendRef` pointing to the `db` service in `k8s-cluster-1`, it will result in a `HTTPRoute` with a `ResolvedRefs` status condition of `BackendNotFound` because service `db` is not present in `k8s-cluster-1`.
```